### ISSUE TITLE
papers/ should be updated

### ISSUE THREAD

**Issue by RichardLitt** [jbenet/ipfs#53]
The current spec in papers is draft 1, and the paper (draft 3) on [your main site](static.benet.ai/t/ipfs.pdf) has some issues where a few of the references are missing. Search for `?` to know what I mean. 

Then again, if you've removed the paper entirely (it's not on ipfs.io), maybe we should just delete this folder?

**Comment by jbenet** [comment]
> few of the references are missing. Search for `?` to know what I mean.

yeah, i hope to fix this in draft 4. i've renamed the issue accordingly. 

>  if you've removed the paper entirely (it's not on ipfs.io)

that's a bug, thanks -- i need to add it back. moved to https://github.com/protocol/websites/issues/5

thanks!

**Comment by RichardLitt** [comment]
np.
